utils/python/CIME/SystemTests/seq.py

```diff
@@ -26,26 +78,27 @@ def run(self):
 
         stop_n      = self._case.get_value("STOP_N")
         stop_option = self._case.get_value("STOP_OPTION")
+        exeroot = self._case.get_value("EXEROOT")
+        cime_model = self._case.get_value("MODEL")
 
         #
         # do an initial run test with default layout
         #
         logger.info("doing a %d %s initial test with default layout" % (stop_n, stop_option))
-        success = SystemTestsCommon._run(self)
 
-        if success:
-            any_changes = False
-            for comp in ['ATM','CPL','OCN','WAV','GLC','ICE','ROF','LND', 'ESP']:
-                any_changes |= self._case.get_value("ROOTPE_%s" % comp) != 0
-                self._case.set_value("ROOTPE_%s" % comp, 0)
-
-            expect(any_changes, "All ROOTPEs were already zero, we aren't testing anything")
+        shutil.copy("%s/%s.exe.SEQ1"%(exeroot,cime_model),
+                    "%s/%s.exe"%(exeroot,cime_model))
+        shutil.copy(os.path.join("LockedFiles", "env_mach_pes.SEQ1.xml"), "env_mach_pes.xml")
+        shutil.copy("env_mach_pes.xml", os.path.join("LockedFiles", "env_mach_pes.xml"))
+        success = SystemTestsCommon._run(self)
```

Comment:
If success is false, the test should probably halt immediately. That's how I did ERS.